### PR TITLE
[SQL-DS-CACHE-47]PlasmaTimeOutWrapper

### PR DESCRIPTION
**Pull request by offthewall123**
## What changes were proposed in this pull request?

PlasmaTimeout wrapper

## How was this patch tested?

e2e

### CODE REVIEW

**Comment by offthewall123** [comment]
@jikunshang

File: Plasma-based-cache/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala [hunk 1]
```diff
@@ -35,7 +35,7 @@ import com.google.common.cache._
 import com.google.common.hash._
 import com.intel.oap.common.unsafe.VMEMCacheJNI
 import org.apache.arrow.plasma
-import org.apache.arrow.plasma.exceptions.{DuplicateObjectException, PlasmaClientException}
+import org.apache.arrow.plasma.exceptions.{DuplicateObjectException, PlasmaClientException, PlasmaGetException}
```
Comment (jikunshang) [inline Plasma-based-cache/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: we have remove this exception `PlasmaGetException`

**Comment by winningsix** [comment]
Quick note. Please associate with a JIRA ID.

File: Plasma-based-cache/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala (hunk 1 above)
Comment (jikunshang) [reply Plasma-based-cache/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: https://github.com/oap-project/sql-ds-cache/pull/18 see changes here

Comment (jikunshang) [reply Plasma-based-cache/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: sorry, should be #39

**Comment by jikunshang** [comment]
I will close this one since we have another fix(#56 ), will re-open this one if it doesn't work.

**Pull request closed without merging**